gm4_garden_variety/data/gm4_garden_variety_trees/functions/load.mcfunction

```diff
@@ -0,0 +1,6 @@
+execute if score gm4 load matches 1 if score gm4_garden_variety load matches 1 run scoreboard players set gm4_garden_variety_trees load 1
+execute unless score gm4 load matches 1 run data modify storage gm4:log queue append value {type:"missing",module:"Garden Variety Trees",require:"Gamemode 4"}
+execute unless score gm4_garden_variety load matches 1 run data modify storage gm4:log queue append value {type:"missing",module:"Garden Variety Trees",require:"Garden Variety"}
```

Comment:
Why is this module split in two, rather than a subfolder in the main namespace?

Reply:
I mirrored how fruiting_trees does it as apple_trees is under its own namespace. It also makes it much easier to understand how an expansion works since you can just copy the _trees namespace.